bears/bears-model.js

```diff
@@ -0,0 +1,45 @@
+const knex = require('knex');
+
+const knexConfig = {
```

Comment:
Lines 1 - 11 could be moved to another file like `knexfile.js` or `dbConfig.js` and then you can use `const db = require('../data/dbConfig');` in all your model files.